### DISCUSSION TITLE
是否可以自定义geosite/geoip的更新地址？

### DISCUSSION THREAD

**Discussion by chenxiccc** [Openwrt-Passwall/openwrt-passwall2#62] [Ideas]
是否可以添加Loyalsoldier的geosite和geoip规则数据？
这个里面有geosite:gfw，可以很好的兼容一部分之前使用gfwlist模式的使用习惯。

比如在“规则管理”中，可以增加一个下拉选择，选择geosite/geoip的数据源使用哪个；
或者，干脆放两个输入框，自定义geosite/geoip的更新地址。

**Comment by chika0801** [comment]
作者默认用的geo数据库就是Loyalsoldier的，在它下载的代码里有。

https://github.com/xiaorouji/openwrt-passwall2/blob/3b190d83adcd71c680d3d2872d8df666619e9b65/luci-app-passwall2/root/usr/share/passwall2/rule_update.lua#L14

**Comment by badgv** [comment]
默认的geosite:geolocation-!cn 感觉是可以直接替代geosite:gfw的

**Reply by chika0801** [reply]
> 默认的geosite:geolocation-!cn 感觉是可以直接替代geosite:gfw的

是的

https://github.com/Loyalsoldier/v2ray-rules-dat#geositedat
作者有详细解释
